Makefile

```diff
@@ -21,6 +21,8 @@ test: generate fmt vet manifests
 # Experimental e2e target
 e2e-sanity:
 	go test tests/base_test.go
+	# vsphere suit
+	go test ./tests/suit -v
```

Comment:
I  think that in the long run it would be better to separate it out to two different jobs - two jobs that for instance take 10 minutes each would complete faster when they run in parallel than a single job that takes 15 minutes

Reply:
yes, its temporary  until we  transition oVirt  tests to the ginkgo framework.